test/Worker.Extensions.Tests/ServiceBus/ServiceBusSessionMessageActionsTests.cs

```diff
@@ -49,6 +50,55 @@ public async Task CanRenewSessionLock()
             await messageActions.RenewSessionLockAsync();
         }
 
+        [Fact]
+        public async Task CanSetAndGetSessionStateWithNull()
```

Comment:
Can you split this test into two different tests:

1. Verifying `null` is returned from `GetSessionStateAsync` when `result.SessionState == null`.
1. Verifying ability to pass `null` to `SetSessionStateAsync`

You shouldn't need any other tests/calls to the client / mock for your set of changes.